x/ibc/03-connection/types/types.proto

```diff
@@ -0,0 +1,137 @@
+syntax = "proto3";
+package cosmos_sdk.x.ibc.connection.v1;
+
+option go_package = "github.com/cosmos/cosmos-sdk/x/ibc/03-connection/types";
+
+import "third_party/proto/gogoproto/gogo.proto";
+import "x/ibc/23-commitment/types/types.proto";
+import "x/ibc/types/types.proto";
+
+// MsgConnectionOpenInit defines the msg sent by an account on Chain A to
+// initialize a connection with Chain B.
+message MsgConnectionOpenInit {
```

Comment:
Do any of these types utilize YAML for their `String` implementation? Keep in mind, YAML tags by default are not snake-cased.